ci/test-nightly.sh

```diff
@@ -27,6 +27,14 @@ maybe_cargo_install() {
   done
 }
 
+# Run program/native/bpf_loader's bench with bpf_c feature
+(
+  set -x
+  cd "programs/native/bpf_loader"
+  echo --- program/native/bpf_loader bench --features=bpf_c
+  cargo +nightly bench --verbose --features="bpf_c" -- --nocapture
```

Comment:
I think `+nightly` is implicit here

Reply:
Maybe but I see this in the rust-nigly docker file:
`cargo +nightly-$date install cargo-cov`